javascripts/htmlBuilder.js

```diff
@@ -31,16 +31,16 @@ var petFood = (function(object) {
 				//ADDS PRICES AND QUANTITIES//
 				var volumes = array[i].types[c].volumes
 				for (var p = 0; p < volumes.length; p++) {
-					content += '<ul class="volumesPrices">'
+					content += '<ul class="volumesPrices">';
 					content += `<li><h4>${volumes[p].name}</h4></li>`;
 					content += `<li>$${volumes[p].price}</li>`;
-					content += '</ul>'
-				}
-				content += '</ul>'
+					content += '</ul>';
```

Comment:
I now wish that chrome wouldn't let me run without the right semicolons, its like not telling someone that they have spinach in their teeth.